src/main/java/com/zendesk/maxwell/schema/SchemaStore.java

```diff
@@ -357,9 +366,50 @@ private void deleteOldSchemas(Long currentSchemaId) throws SQLException {
 
 		Long toDelete = currentSchemaId - maxSchemas; // start with the highest numbered ID to delete, work downwards until we run out
 		while ( toDelete > 0 && schemaExists(toDelete) ) {
-			new SchemaStore(connection, serverID, toDelete).destroy();
+			new SchemaStore(connection, serverID, toDelete).delete();
 			toDelete--;
 		}
 	}
 
+	/*
+		for the time being, when we detect other schemas we will simply wipe them down.
+		in the future, this is our moment to pick up where the master left off.
```

Comment:
well we only soft delete because of the replication pressure.  it's functionally the same as a hard delete